test/steps/transactions/3_then.js

```diff
@@ -18,6 +18,52 @@ import {
 	getNumbers,
 	getTransactionCreatorFunctionNameByType,
 } from '../utils';
+import {
+	prependPlusToPublicKeys,
+	prependMinusToPublicKeys,
+} from '../../../src/utils/helpers';
+
+export function itShouldCreateACastVoteTransactionWithThePassphraseTheSecondPassphraseAndThePublicKeysPrependedWithTheCorrectModifier() {
+	const {
+		passphrase, secondPassphrase, votePublicKeys, unvotePublicKeys,
+	} = this.test.ctx;
+	const votes = prependPlusToPublicKeys(votePublicKeys);
+	const unvotes = prependMinusToPublicKeys(unvotePublicKeys);
```

Comment:
Source code shouldn't be used as helpers in step definitions

Reply:
This will be solved with including these functions into lisk-js https://github.com/LiskHQ/lisk-js/issues/486